crates/rattler_conda_types/src/channel/mod.rs

```diff
@@ -27,13 +27,49 @@ pub struct ChannelConfig {
     ///
     /// The default value is: <https://conda.anaconda.org>
     pub channel_alias: Url,
+
+    /// An optional configuration for the repodata of the channel.
+    ///
+    /// This *SHOULD* be used to configure how metadata is fetched from the channel.
+    ///
+    pub repodata_options: Option<ChannelRepodataConfig>,
+}
+
+fn _serde_is_true(value: &bool) -> bool {
```

Comment:
Prefixing with _ would mean its unused. I assume you want to make it private to the module, if you dont add "pub" that should be fine.